packages/backend/src/Models/Elections.ts

```diff
@@ -89,6 +90,7 @@ export default class ElectionsDB implements IElectionStore {
         let query = this._postgresClient
             .selectFrom(tableName)
             .where('head', '=', true)
+            .where("is_public", "=", true)
```

Comment:
Can you remove this one?

getElections will also needs to be able to query private elections